ios/PurchasesHybridCommon/Podfile

```diff
@@ -15,3 +16,31 @@ target 'PurchasesHybridCommon' do
   end
 
 end
+
```

Comment:
a lot of boilerplate in this one, but it can be summed up as: 
- `PurchasesHybridCommon` uses `PurchasesHybridCommonSwift`
- `ObjCApiTester` uses `PurchasesHybridCommon` and `PurchasesHybridCommonSwift`
- They use local paths because we're not going to actually publish `PurchasesHybridCommonSwift` as a pod this time around, since it won't live long enough for that